development/build/transforms/utils.js

```diff
@@ -1,11 +1,17 @@
 const { ESLint } = require('eslint');
 const eslintrc = require('../../../.eslintrc.js');
 
-// We don't want linting to fail for purely stylistic reasons.
-eslintrc.rules['prettier/prettier'] = 'off';
-// Sometimes we use `let` instead of `const` to assign variables depending on
-// the build type.
-eslintrc.rules['prefer-const'] = 'off';
+eslintrc.overrides.forEach((override) => {
```

Comment:
Changes are necessary to this file (which is part of the build system) because we run ESLint during the build system. To do this we pull in the ESLint configuration object from `.eslintrc.js` and we explicitly modify that object to disable some rules as this would cause failures which we are not interested in. Because ALL ESLint rules are now in `overrides`, I have to change how these modifications work.